gcloud/bigquery/job.py

```diff
@@ -965,6 +971,8 @@ def _populate_config_resource(self, configuration):
             configuration['priority'] = self.priority
         if self.use_query_cache is not None:
             configuration['useQueryCache'] = self.use_query_cache
+        if self.user_defined_function_resources is not None:
+            configuration['userDefinedFunctionResources'] = self.user_defined_function_resources
```

Comment:
pep8 doesn't like the long line.  Please wrap to < 79 characters (e.g., open a parenthesis after the `=` and wrap to the next line).